migrator/migrations/n_57_to_n_58_postgres_create_policy_category_edges/migration.go

```diff
@@ -0,0 +1,139 @@
+package n_57_to_n_58_postgres_create_policy_category_edges
```

Comment:
I think I would rather this be m_170_to_m_171 and the create edge be m_171_to_m_172.  Or even just a single migration.  ACSCS has previously talked about using more rapid builds.  If they were to have done that then they could have already executed m_168_to_m_169 in theory.  If that were to be the case then their next upgrade would not execute this one.  It also makes the math fuzzy.

Reply:
Aah yes, this needs to be removed. This was prior to our m and n discussion.